bin/riemann-rabbitmq

```diff
@@ -17,6 +17,7 @@ class Riemann::Tools::Rabbitmq
   opt :monitor_pass, 'RabbitMQ monitoring user password', type: :string
   opt :monitor_port, 'RabbitMQ monitoring port', type: :int, default: 15672
   opt :monitor_host, 'RabbitMQ monitoring host', type: :string, default: "localhost"
+  opt :monitor_ssl, 'RabbitMQ ssl', type: :string, default: "false"
```

Comment:
Yeah, a protocol string might be easiest but too verbose. I it is a string, why not expose "url" to provide custom schemes overriding the rest?